Do final edits to proposal (#3)
cct-datascience/volcalc-isc-proposal: proposal/success.Rmd

- Can the package be installed from GitHub? (yes/no)
- Can the package be installed from r-universe? (yes/no)
- Is there sufficient test coverage? (at least 90%)
- Are there correctness tests for volatility predictions including comparisons with measured values (± some tolerance) (yes/no)
- Is the package passing `R CMD check` on Linux, macOS, and Windows using CI? (yes/no)
- Has the package been tested by users other than the developers? (yes/no)
- Does the package have a vignette that is easy to follow? (yes/no)
- Is code archived on Zenodo and a DOI associated with the package citation? (yes/no)
- Can chemical representation types be used as input? (molfile for success; InChI, SMILES, and possibly more for reach goal)

## Future work
<!-- 
How could this be extended / developed in the future by yourself and/or the community in general?
-->

> - As a handful of functional groups relevant to volatility were not able to be measured using existing methods, the community who creates these methods could expand to these groups

Reviewer: Feel free to reject suggestion if I'm not getting your meaning.

```suggestion
- The contribution of some functional groups to volatility were not able to be measured using existing methods. As these analytical methods improve, the algorithm in `volcalc` can be updated.
```

> - The algorithm used in the package only calculates relative volatility under standard conditions; the algorithm could be added to or expanded to calculate volatility itself and for additional environmental conditions

Reviewer: ```suggestion
- `volcalc` only calculates relative volatility under standard conditions; functionality to specify environmental conditions could be added.
```

> - As additional chemical compound databases are used as inputs to `volcalc`, these should be documented and either added to this package or created as standalone packages that link to `volcalc`

Reviewer: I would just remove this.  I don't think we want to get into the API wrapper game
```suggestion
```

> - As this package is used increasingly by researchers, they will discover other edge cases or unusual compounds that the package can be improved to accomodate

Reviewer: I don't think this is necessary here.  Could be part of the proposal somewhere else to mention that the Meredith lab and CCT will continue maintainance and development after this grant instead.
```suggestion
```
